ecosystem/node-checker/src/args.rs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{metric_evaluator::StateSyncMetricsEvaluatorArgs, runner::BlockingRunnerArgs};
+use clap::Parser;
+use once_cell::sync::Lazy;
+use url::Url;
+
+pub const DEFAULT_METRICS_PORT: u16 = 9101;
+pub const DEFAULT_API_PORT: u16 = 8080;
+pub const DEFAULT_NOISE_PORT: u16 = 6180;
+
+pub static DEFAULT_METRICS_PORT_STR: Lazy<String> =
+    Lazy::new(|| format!("{}", DEFAULT_METRICS_PORT));
+pub static DEFAULT_API_PORT_STR: Lazy<String> = Lazy::new(|| format!("{}", DEFAULT_API_PORT));
+pub static DEFAULT_NOISE_PORT_STR: Lazy<String> = Lazy::new(|| format!("{}", DEFAULT_NOISE_PORT));
+
+#[derive(Clone, Debug, Parser)]
+#[clap(author, version, about, long_about = None)]
+pub struct Args {
+    /// Whether to enable debug logging or not. This is a shortcut for the
+    /// standard env logger configuration via env vars
+    #[clap(short, long)]
+    pub debug: bool,
```

Comment:
Normally I have this if the tool is something you run once, like a CLI, and otherwise not. While at this commit this falls into the latter bucket, in later commits I add other subcommands that you would run once (e.g. `nhc configuration create`), in which case I think `-d` is a nice helper vs having to set a couple of env vars with every CLI invocation.

Reply:
I guess the question in my mind is "Why would the user want debug logs?" to which the only answer I can come up with is "for debugging/development"?. If you suspect regular users (i.e., those who aren't developers) want to see debug logs, it feels to me like we should just log relevant information at a higher level as opposed to adding a flag that explicitly sets debug logs. One could wonder, why not a flag for all debug levels?

But, this isn't a blocker 😄

Reply:
Ok yeah I think from that perspective it makes sense, I'll remove the flag 👍